code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -112,22 +112,27 @@
 		icon_state = "[initial(icon_state)]0"
 
 
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/empty
+	name = "autoinjector"
+	preloaded_reagents = list("inaprovaline" = 0)
+	#Yes I know this is stupid, but it's a lot easier than making the parent empty and fixing all the fallout from that for now.
```

Comment:
Wrong comment prefix. Use `//` or `/* */` (for multi-line or block comments) when you want to write a comment in DM code.
```suggestion
	// Yes I know this is stupid, but it's a lot easier than making the parent empty and fixing all the fallout from that for now.
```